docs/README.rst

```diff
@@ -1,85 +1,132 @@
+   :target: https://github.com/semantic-release/semantic-release
+
 Linux logical volume management (LVM2) state API. 
 
 note:: The `lvm.conf(5)` is indirectly managed via LVM profiles.
 
-note:: See the full `Salt Formulas installation and usage instructions
-    <http://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+.. contents:: **Table of Contents**
+   :depth: 1
+
+General notes
+-------------
+
+See the full `SaltStack Formulas installation and usage instructions
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+
+If you are interested in writing or contributing to formulas, please pay attention to the `Writing Formula Section
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#writing-formulas>`_.
+
+If you want to use this formula, please pay attention to the ``FORMULA`` file and/or ``git tag``,
+which contains the currently released version. This formula is versioned according to `Semantic Versioning <http://semver.org/>`_.
+
+See `Formula Versioning Section <https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#versioning>`_ for more details.
+
+Todo
+^^^^
+- global filter support is important
+- test some advanced LV/RAID scenarios
+- file systems mngt
+
+Good pillar data
+^^^^^^^^^^^^^^^^
+Bad configuration causes problems. Sanity check pillar data when troubleshooting "``unable to``" state failures.
+
+OS families
+^^^^^^^^^^^
+All Linux distributions supported.
+
+Contributing to this repo
+-------------------------
+
+**Commit message formatting is significant!!**
+
+Please see :ref:`How to contribute <CONTRIBUTING>` for more details.
 
 Available Meta states
-======================
+---------------------
 
 .. contents::
     :local:
 
 ``lvm``
---------
+^^^^^^^
 Meta-state to run all states in sequence: 'install', 'profiles', 'files', 'pv', 'vg', and 'lv'.
 
 ``lvm.profiles``
---------------
+^^^^^^^^^^^^^^^^
 Meta-state to manage lvm profiles in sequence: 'remove', followed by 'create'.
 
 ``lvm.files``
------------
+^^^^^^^^^^^^^
 Meta-state to run loopback file device states in sequence: 'remove', followed by 'create'. Included by `lvm.pv` state.
 
 ``lvm.pv``
------------
+^^^^^^^^^^
 Meta-state to run physical volume (PV) states in sequence: 'remove', 'change', 'resize', 'move', and finally 'create'.
 
 ``lvm.vg``
---------------
+^^^^^^^^^^
 Meta-state to run all volume group states in sequence: 'cfgbackup', 'import', 'remove', 'change' 'reduce', 'extend', 'split', 'merge', 'rename', 'create', 'export' & 'cfgrestore'.
 
 ``lvm.lv``
--------------
+^^^^^^^^^^
 Meta-state to run all logical volume states in sequence: Order 'remove', 'change', 'reduce', 'extend', 'rename', 'create', 'convert', and 'create' again.
 
 
 Available substates
-===================
+-------------------
 
 .. contents::
     :local:
 
-``lvm.remove``
-------------
+``lvm.clean``
+^^^^^^^^^^^^^
 Remove lvm2 software.
 
```

Comment:
There is a duplicate `lvm.clean` on the rendered [README](https://github.com/saltstack-formulas/lvm-formula/blob/454f0a1a5b9d52fe19009298ffc4f1f56fd3afca/docs/README.rst#available-substates).